lib/Documentable/CLI.pm6

```diff
@@ -120,10 +121,14 @@ package Documentable::CLI {
         Bool :a(:$all)             = False                    #= Equivalent to -t -p -s -i --search-index
     ) {
         my $beginning = now; # to measure total time
-        if (!"./html".IO.e || !"./assets".IO.e || !"./template".IO.e and $v) {
-            note q:to/END/;
-                (warning) html and/or assets and/or template directories
-                cannot be found. You can get the defaults by executing:
+	my $asset-dir-missing = @default-asset-dirs.map({!.IO.e}).any;
+        if ($asset-dir-missing and $v) {
+            note qq:to/END/;
+                (warning) one of the following directories cannot be found:
+
+		    @default-asset-dirs.join(', ')
+
+		You can get the defaults by executing:
```

Comment:
The indenting here is also odd and doesn't match what I have in my local copy.  Perhaps GitHub didn't handle the force push properly?